TS02-Convert-to-Date-Time-Class-POSIX.Rmd

```diff
@@ -86,22 +87,12 @@ library(lubridate)  #work with dates
 #set working directory to ensure R can find the file we wish to import
 #setwd("working-dir-path-here")
 
```

Comment:
this is great! looks like you pulled imported the data. We need to do that because these lessons will also be standalone. (correct me if i missed something!). 

Thank you for all of the great edits!